src/provider/application.js

```diff
@@ -3,16 +3,17 @@ import { fixedTimeCompare } from '../lib/string';
 import { EventEmitter } from 'events';
 import URI from '../lib/uri';
 import logger from '../lib/logger';
-import { calculateHeight, calculateWidth } from '../lib/dimension';
+import { getOffsetToBody, calculateHeight, calculateWidth } from '../lib/dimension';
 import MutationObserver from 'mutation-observer';
 
 
 /** Application class which represents an embedded application. */
 class Application extends EventEmitter {
-  init({ acls = [], secret = null, onReady = null }) {
+  init({ acls = [], secret = null, onReady = null, targetSelectors = [] }) {
```

Comment:
this doesn't need to be an array, `querySelectorAll` can take multiple selectors comma separated which means we can execute just one query.

Reply:
Please add documentation to this function on what targetSelectors is